ParisBNF/abb/BNFabb2.xml

```diff
@@ -0,0 +1,308 @@
+                                 <title type="complete" ref="LIT1882MarkGo#BiographyMark"/>
+                                 <textLang mainLang="gez"/>
+                              </msItem>
+                                </msItem>
+
+                              <msItem xml:id="ms_i1.4">
+                                 <locus from="84r"/>
+                                 <title type="complete" ref="LIT1812GospelLuke"/>
+                                 <textLang mainLang="gez"/>
+
+                                 <msItem xml:id="ms_i1.4.1">
+                                    <locus from="84r"/>
+                                    <title type="complete" ref="LIT2713Luke"/>
+                                    <textLang mainLang="gez"/>
+                                 </msItem>
+
+                                 <msItem xml:id="ms_i1.4.2">
+                                   <title type="complete" ref="LIT1812GospelLuke#BiographyLuke"/>
+                                    <textLang mainLang="gez"/>
+                                 </msItem>
+                                    </msItem>
+
+                           <msItem xml:id="ms_i1.5">
+                             <locus from="131r"/>
+                              <title type="complete" ref="LIT1693John"/>
+                              <textLang mainLang="gez"/>
+
+                              <msItem xml:id="ms_i1.5.1">
+                                 <locus from="161ra" to="217vb"/>
+                                 <title type="complete" ref="LIT2715John"/>
+
+                                        </msItem>
+
+                              <msItem xml:id="ms_i1.5.2">
+                                 <locus from="131r"/>
+                                 <title type="complete" ref="LIT1693John#BiographyJohn"/>
+                                 <textLang mainLang="gez"/>
+                                         </msItem>
+                                      </msItem>
+
+                         </msContents>
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <material key="parchment"/>
+                        </support>
+                        <extent>
+                          <measure unit="leaf">166</measure>
+                          <note><bibl>
+                             <ptr target="bm:Abbadie1859Cat"/>
+                             <citedRange unit="page">2</citedRange></bibl>
+                             indicates the presence of only 162 folia.</note>
+                             <measure unit="leaf" type="blank">4<locus target="#1 #2 #7 #57"></locus>
+                          </measure>
+                               <dimensions type="outer">
+                              <height unit="mm">276</height>
+                              <width unit="mm">241</width>
+                              <depth unit="mm"></depth>
```

Comment:
I think that all empty elements, here and in layout and binding, could be deleted. If the images will be available at some point, they can be added.

Reply:
done